docs/guides/providing-terms.md

```diff
@@ -1,23 +1,22 @@
-Setup your terms of use
-===
+# Setup
```

Comment:
"Setup" sounds unfitting in my opinion as this file is explicitly only about setting up legal files like the terms.

Reply:
Do you have any proposal?

Maybe "legal setup" or "legal terms setup"?

Reply:
Since this falls into the how-to guide category of documentation, it should be "How to set up your terms of use"

Reply:
I've changed the title.

Reply:
I've changed the title according to @SISheogorath's proposal.